projects/lf-documentation/src/app/getting-started-npm-example/getting-started-npm-example.component.html

```diff
@@ -0,0 +1,72 @@
+<h3 id="npm-example">Example: <code>lf-checklist</code> In an Angular Project Via NPM
+</h3>
+<p>In the example below, both the implementation and types are provided via <code>@laserfiche/lf-ui-components/lf-checklist</code>.</p>
+<ol>
```

Comment:
You don't need to put `Via NPM` in all the titles since this page is all NPM

There could be a title though
`Examples using the NPM package`